lib/pbench/test/functional/agent/cli/commands/tools/test_clear_tools.py

```diff
@@ -208,5 +206,151 @@ def test_clear_tools_test70(monkeypatch, agent_config, pbench_run, pbench_cfg):
+    default_group = pbench_run / "tools-v1-default"
+    default_group.mkdir(parents=True)
+    fubar_default_host = default_group / "fubar6.example.com"
+    fubar_default_host.mkdir(parents=True, exist_ok=True)
+    pidstat_default_tool = fubar_default_host / "pidstat"
+    pidstat_default_tool.touch()
+
+    another_group = pbench_run / "tools-v1-another"
+    another_group.mkdir(parents=True)
+    fubar_another_host = another_group / "fubar7.example.com"
+    fubar_another_host.mkdir(parents=True, exist_ok=True)
+    pidstat_another_tool = fubar_another_host / "pidstat"
+    pidstat_another_tool.touch()
+
+    command = ["pbench-clear-tools", "--group=default,another"]
+    out, err, exitcode = pytest.helpers.capture(command)
+    assert b"" == out
+    assert default_group.exists() is True
+    assert fubar_default_host.exists() is False
+    assert another_group.exists() is False
+    assert fubar_another_host.exists() is False
+
+
+def test_clear_tools_test72(monkeypatch, agent_config, pbench_run, pbench_cfg):
+    """ Attempt to clear the tool groups, by name, separated by comma.
+    However one of the tool group name is wrong"""
+    monkeypatch.setenv("_PBENCH_AGENT_CONFIG", str(pbench_cfg))
+
+    another_group = pbench_run / "tools-v1-another"
+    another_group.mkdir(parents=True)
+    fubar_another_host = another_group / "fubar7.example.com"
+    fubar_another_host.mkdir(parents=True, exist_ok=True)
+    pidstat_another_tool = fubar_another_host / "pidstat"
+    pidstat_another_tool.touch()
+
+    command = ["pbench-clear-tools", "--group=another,wrong"]
+    out, err, exitcode = pytest.helpers.capture(command)
+    assert b"" == out
+    assert (
+        b'Removed "pidstat" from host "fubar7.example.com" in tools group '
+        b'"another' in err
+    )
+    assert b'pbench-clear-tools: invalid --group option "wrong"' in err
+    assert another_group.exists() is False
+    assert fubar_another_host.exists() is False
+
+
+def test_clear_tools_test73(monkeypatch, agent_config, pbench_run, pbench_cfg):
+    """ Attempt to clear the tools, by name, separated by comma"""
+    monkeypatch.setenv("_PBENCH_AGENT_CONFIG", str(pbench_cfg))
+    default_group = pbench_run / "tools-v1-default"
+    default_group.mkdir(parents=True)
+    fubar6_host = default_group / "fubar6.example.com"
+    fubar6_host.mkdir(parents=True, exist_ok=True)
+    pidstat6_tool = fubar6_host / "pidstat"
+    pidstat6_tool.touch()
+    mpstat6_tool = fubar6_host / "mpstat"
+    mpstat6_tool.touch()
+    iostat6_tool = fubar6_host / "iostat"
+    iostat6_tool.touch()
+
+    command = ["pbench-clear-tools", "--name=pidstat,mpstat"]
+    out, err, exitcode = pytest.helpers.capture(command)
+    assert b"" == out
+    assert b"" in err
+    assert pidstat6_tool.exists() is False
+    assert mpstat6_tool.exists() is False
+    assert iostat6_tool.exists() is True
+    assert fubar6_host.exists() is True
```

Comment:
Given that `iostat6_tool` exists and is in `fubar6_host`, it doesn't seem like we need to check for the existence of `fubar6_host`.